hack/lib/util.sh

```diff
@@ -136,14 +136,21 @@ kube::util::gen-doc() {
   # create the list of generated files
   ls "${tmpdir}" | LC_ALL=C sort > "${tmpdir}/.files_generated"
 
-  # remove all old generated file from the destination
   while read file; do
+    # Add analytics link to generated .md files
+    if [[ "${file}" == *".md"* ]]; then
```

Comment:
I'd skip the quotes on the right side `[[ "${file}" == *.md ]]`, also why the `*` after `.md`? What kind of file will match that?